examples/environment-guide/backend-service.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1beta3
+kind: Service
+metadata:
+  creationTimestamp: null
+  name: backend-srv
```

Comment:
I'd add the `type: backend-type` label to the service, which is the practice we recommend and would make delete simpler.